covid.py

```diff
@@ -1 +1,67 @@
-#this is comment
+import argparse
+import json
+import requests
+
+
+def parser():
+    my_parser = argparse.ArgumentParser(
+        description="By giving name of country you will get "
+        "percentage of people recovered and still sick from all population"
+    )
+    my_parser.add_argument(
+        "--country",
+        action="store",
+        required=True,
+        help="Name of country",
+    )
+    return my_parser.parse_args()
+
+
+def computing_answer(population, confirmed_cases, recovered_people, dead_people):
+    percent_of_recovered = round(100 * recovered_people / population, 1)
+    percent_of_sick = round(
+        100 * (confirmed_cases - recovered_people - dead_people) / population, 1
+    )
+    print(
```

Comment:
I'm not a fan. You're violating a bit [SRP](https://en.wikipedia.org/wiki/Single-responsibility_principle) principle. Looking at the `computing_answer` function I expect it to return `computed_answer`. Your function is bugged with the side effect (it prints data)